lib/Wallet/WalletRpcServer.cpp

```diff
@@ -139,6 +139,7 @@ void wallet_rpc_server::processRequest(const CryptoNote::HttpRequest& request, C
             { "stop_wallet"      , makeMemberMethod(&wallet_rpc_server::on_stop_wallet)       },
             { "reset"            , makeMemberMethod(&wallet_rpc_server::on_reset)             },
             { "get_payments"     , makeMemberMethod(&wallet_rpc_server::on_get_payments)      },
+			{ "get_messages"	 , makeMemberMethod(&wallet_rpc_server::on_get_messages)	  },
```

Comment:
Indent is too big.
```suggestion
            { "get_messages"	 , makeMemberMethod(&wallet_rpc_server::on_get_messages)	  },
```

Reply:
Please, always use spaces for indentation. **Never use tabs!** This line is currently indented with tabs.